app/src/main/java/fr/free/nrw/commons/AboutActivity.java

```diff
@@ -47,22 +49,61 @@ public void launchFacebook(View view) {
             intent.setPackage("com.facebook.katana");
             startActivity(intent);
         } catch (Exception e) {
-            startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://www.facebook.com/" + "1921335171459985")));
+            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
+            builder.setToolbarColor(ContextCompat.getColor(this, R.color.primaryColor));
+            builder.setSecondaryToolbarColor(ContextCompat.getColor(this, R.color.primaryDarkColor));
+            builder.setExitAnimations(this, android.R.anim.slide_in_left, android.R.anim.slide_out_right);
+            CustomTabsIntent customTabsIntent = builder.build();
+            customTabsIntent.intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+            customTabsIntent.launchUrl(this, Uri.parse("https://www.facebook.com/" + "1921335171459985"));
         }
-
     }
 
     @OnClick(R.id.github_launch_icon)
     public void launchGithub(View view) {
-
-        Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/commons-app/apps-android-commons\\"));
-        startActivity(browserIntent);
+        CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
```

Comment:
Instead of defining this in the `Activity` class itself, extract it out to a util class. That would avoid duplication. 

```
Utils.handleWebUrl(context, url);
```

This will also help if we later on decide how to handle a web url. That won't require a change in multiple files. :)

Reply:
Ok sir thanks. I will do it.

Reply:
Sir I had added the method and tested it.